src/hist/plot.py

```diff
@@ -20,7 +34,42 @@
     raise
 
 
-__all__ = ("histplot", "hist2dplot", "plot2d_full", "plot_pull", "plot_pie")
+class FitResultArtists(NamedTuple):
+    line: matplotlib.lines.Line2D
+    errorbar: matplotlib.container.ErrorbarContainer
+    band: matplotlib.collections.PolyCollection
+
+
+class RatioErrorbarArtists(NamedTuple):
+    line: matplotlib.lines.Line2D
+    errorbar: matplotlib.container.ErrorbarContainer
+
+
+class RatioBarArtists(NamedTuple):
+    line: matplotlib.lines.Line2D
+    dots: matplotlib.collections.PathCollection
+    bar: matplotlib.container.BarContainer
+
+
+class PullArtists(NamedTuple):
+    bar: matplotlib.container.BarContainer
+    patch_artist: List[matplotlib.patches.Rectangle]
+
+
+MainAxisArtists = Union[FitResultArtists, Hist1DArtists]
+
+RatioArtists = Union[RatioErrorbarArtists, RatioBarArtists]
+RatiolikeArtists = Union[RatioArtists, PullArtists]
+
+
+__all__ = (
+    "histplot",
+    "hist2dplot",
+    "plot2d_full",
+    "plot_ratio",
+    "plot_pull",
+    "plot_pie",
+)
```

Comment:
Minor, but I"d generally place `__all__` as close to the top as reasonably allowed (just below the imports), since it's handy for users looking in a file to see what is publicly available. (I can change this later, so no need to change)

Reply:
I can get this now. :)

Reply:
Okay moved them.